js/apps/admin-ui/src/client-scopes/details/ScopeForm.tsx

```diff
@@ -143,10 +166,18 @@ export const ScopeForm = ({ clientScope, save }: ScopeFormProps) => {
             label={t("protocol")}
             labelIcon={t("protocolHelp")}
             controller={{ defaultValue: providers[0] }}
-            options={providers.map((option) => ({
-              key: option,
-              value: getProtocolName(t, option),
-            }))}
+            options={providers
+              .filter((option) => {
+                if (option === OID4VC_PROTOCOL) {
+                  return isOid4vcEnabled();
+                }
+                return true;
+              })
```

Comment:
This is a matter of taste, but I would shorten this to:

```suggestion
              .filter((option) => option === OID4VC_PROTOCOL ? isOid4vcEnabled : true)
```